packages/webpack/template-webpack-plugin/src/LynxTemplatePlugin.ts

```diff
@@ -117,6 +117,7 @@ export interface TemplateHooks {
     template: Buffer;
     outputName: string;
     mainThreadAssets: Asset[];
+    cssChunks: Asset[];
```

Comment:
I don't think we actually need this since there are enough information in `finalEncodeOptions`. It's fine to add this now, but I would like to make a refactor and avoid having these extra fields before #1098 and #1035.

Reply:
E.g.:

```js
          const { buffer } = await hooks.encode.promise({
            encodeOptions: {
              ...args.finalEncodeOptions,
              lepusCode: {
                root: undefined,
                lepusChunk: {},
              },
              manifest: {},
              customSections: {},
            },
          });
```

Reply:
In `finalEncodeOptions`, we cannot retrieve the CSS chunk filename; it only contains the `cssMap` and `cssSource` fields.

<img width="558" alt="image" src="https://github.com/user-attachments/assets/032e7791-9fed-4746-8756-32091d46a805" />